### ISSUE TITLE
unfortunately something went wrong

### ISSUE THREAD

**Issue by abhishek-techs** [jitsi/jicofo#817]
Hello All, 

I have installed jitsi-meet  according to the following article

https://jitsi.github.io/handbook/docs/devops-guide/devops-guide-scalable
 
I have installed the following package on one server
* `nginx`
* `prosody`
* `jicofo`
* `jitsi-meet-web`
* `jitsi-meet-prosody`
* `jitsi-meet-web-config`
* `openjdk-8`
-------------------------------------------------------------------------
 And in another server, I have installed jitst-videobridge-2

----------------------------------------------------------------------------
Now, The issue is whenever I am creating the jitsi video call room sometimes it's showing "unfortunately something went wrong" and sometimes it's work I don't know why I am facing that issue. Further, I didn't see any issue in the server log but there is an issue in the browser console  and it' following

---------------
Logger.js:154 2021-10-06T20:10:26.331Z [JitsiMeetJS.js] <Object.getGlobalOnErrorHandler>: UnhandledError: Focus error, retry after 1000 Script: null Line: null Column: null StackTrace: Error: Focus error, retry after 1000

at l._allocateConferenceFocusError (https://vid-test.hexafair.com/libs/lib-jitsi-meet.min.js?v=5307:10:178666)

at https://vid-test.hexafair.com/libs/lib-jitsi-meet.min.js?v=5307:10:177717

at w.Handler.handler (https://vid-test.hexafair.com/libs/lib-jitsi-meet.min.js?v=5307:1:32187)

at w.Handler.run (https://vid-test.hexafair.com/libs/lib-jitsi-meet.min.js?v=5307:1:27486)

at https://vid-test.hexafair.com/libs/lib-jitsi-meet.min.js?v=5307:1:35924

at Object.forEachChild (https://vid-test.hexafair.com/libs/lib-jitsi-meet.min.js?v=5307:1:19148)

at w.Connection._dataRecv (https://vid-test.hexafair.com/libs/lib-jitsi-meet.min.js?v=5307:1:35773)

at O.Bosh._onRequestStateChange (https://vid-test.hexafair.com/libs/lib-jitsi-meet.min.js?v=5307:1:55758)

o @ Logger.js:154

getGlobalOnErrorHandler @ JitsiMeetJS.js:545

window.onerror @ middleware.js:110

callErrorHandler @ GlobalOnErrorHandler.js:61

l._allocateConferenceFocusError @ moderator.js:349

(anonymous) @ moderator.js:276

(anonymous) @ strophe.umd.js:2716

run @ strophe.umd.js:1875

(anonymous) @ strophe.umd.js:3157

forEachChild @ strophe.umd.js:830

_dataRecv @ strophe.umd.js:3146

_onRequestStateChange @ strophe.umd.js:5012

XMLHttpRequest.send (async)

d @ strophe.umd.js:5123

_processRequest @ strophe.umd.js:5137

_throttledRequestHandler @ strophe.umd.js:5290

_onIdle @ strophe.umd.js:4901

_onIdle @ strophe.umd.js:3881

(anonymous) @ strophe.umd.js:5257

setTimeout (async)

_send @ strophe.umd.js:5256

send @ strophe.umd.js:2583

sendIQ @ strophe.umd.js:2740

info @ strophe.disco.js:123

(anonymous) @ Caps.js:213

_getDiscoInfo @ Caps.js:212

getFeaturesAndIdentities @ Caps.js:200

(anonymous) @ xmpp.js:439

_processDiscoInfoIdentities @ xmpp.js:413

(anonymous) @ xmpp.js:304

Promise.then (async)

connectionHandler @ xmpp.js:298

_stropheConnectionCb @ XmppConnection.js:308

_changeConnectStatus @ strophe.umd.js:3011

_onSessionResultIQ @ strophe.umd.js:3705

run @ strophe.umd.js:1875

(anonymous) @ strophe.umd.js:3157

forEachChild @ strophe.umd.js:830

_dataRecv @ strophe.umd.js:3146

_onRequestStateChange @ strophe.umd.js:5012

XMLHttpRequest.send (async)

d @ strophe.umd.js:5123

_processRequest @ strophe.umd.js:5137

_throttledRequestHandler @ strophe.umd.js:5290

_onIdle @ strophe.umd.js:4901

_onIdle @ strophe.umd.js:3881

(anonymous) @ strophe.umd.js:5257

setTimeout (async)

_send @ strophe.umd.js:5256

send @ strophe.umd.js:2583

_establishSession @ strophe.umd.js:3675

_onResourceBindResultIQ @ strophe.umd.js:3644

run @ strophe.umd.js:1875

(anonymous) @ strophe.umd.js:3157

forEachChild @ strophe.umd.js:830

_dataRecv @ strophe.umd.js:3146

_onRequestStateChange @ strophe.umd.js:5012

XMLHttpRequest.send (async)

d @ strophe.umd.js:5123

_processRequest @ strophe.umd.js:5137

_throttledRequestHandler @ strophe.umd.js:5290

_onIdle @ strophe.umd.js:4901

_onIdle @ strophe.umd.js:3881

(anonymous) @ strophe.umd.js:5257

setTimeout (async)

_send @ strophe.umd.js:5256

send @ strophe.umd.js:2583

bind @ strophe.umd.js:3601

_onStreamFeaturesAfterSASL @ strophe.umd.js:3562

_onStreamFeaturesAfterSASL @ strophe.stream-management.js:230

s @ strophe.umd.js:3517

(anonymous) @ strophe.umd.js:3523

run @ strophe.umd.js:1875

(anonymous) @ strophe.umd.js:3157

forEachChild @ strophe.umd.js:830

_dataRecv @ strophe.umd.js:3146

_onRequestStateChange @ strophe.umd.js:5012

XMLHttpRequest.send (async)

d @ strophe.umd.js:5123

_processRequest @ strophe.umd.js:5137

_throttledRequestHandler @ strophe.umd.js:5290

_onIdle @ strophe.umd.js:4901

_onIdle @ strophe.umd.js:3881

(anonymous) @ strophe.umd.js:2769

setTimeout (async)

_sendRestart @ strophe.umd.js:2768

_sasl_success_cb @ strophe.umd.js:3529

run @ strophe.umd.js:1875

(anonymous) @ strophe.umd.js:3157

forEachChild @ strophe.umd.js:830

_dataRecv @ strophe.umd.js:3146

_onRequestStateChange @ strophe.umd.js:5012

XMLHttpRequest.send (async)

d @ strophe.umd.js:5123

_processRequest @ strophe.umd.js:5137

_throttledRequestHandler @ strophe.umd.js:5290

_onIdle @ strophe.umd.js:4901

_onIdle @ strophe.umd.js:3881

(anonymous) @ strophe.umd.js:5257

setTimeout (async)

_send @ strophe.umd.js:5256

send @ strophe.umd.js:2583

_sasl_challenge_cb @ strophe.umd.js:3389

run @ strophe.umd.js:1875

(anonymous) @ strophe.umd.js:3157

forEachChild @ strophe.umd.js:830

_dataRecv @ strophe.umd.js:3146

_onRequestStateChange @ strophe.umd.js:5012

XMLHttpRequest.send (async)

d @ strophe.umd.js:5123

_processRequest @ strophe.umd.js:5137

_throttledRequestHandler @ strophe.umd.js:5290

_onIdle @ strophe.umd.js:4901

_onIdle @ strophe.umd.js:3881

(anonymous) @ strophe.umd.js:5257

setTimeout (async)

_send @ strophe.umd.js:5256

send @ strophe.umd.js:2583

_attemptSASLAuth @ strophe.umd.js:3364

authenticate @ strophe.umd.js:3319

_connect_cb @ strophe.umd.js:3271

_onRequestStateChange @ strophe.umd.js:5012

XMLHttpRequest.send (async)

d @ strophe.umd.js:5123

_processRequest @ strophe.umd.js:5137

_throttledRequestHandler @ strophe.umd.js:5290

_connect @ strophe.umd.js:4586

connect @ strophe.umd.js:2368

_interceptConnectArgs @ strophe.stream-management.js:224

connect @ XmppConnection.js:264

_connect @ xmpp.js:517

connect @ xmpp.js:626

c.connect @ JitsiConnection.js:61

e @ connection.js:52

(anonymous) @ connection.js:198

b @ connection.js:123

_onLogin @ LoginDialog.js:158

_onSubmit @ AbstractDialog.js:117

_onSubmit @ StatelessDialog.js:259

(anonymous) @ usePlatformLeafEventHandler.js:60

f @ react-dom.production.min.js:15

_ @ react-dom.production.min.js:15

(anonymous) @ react-dom.production.min.js:16

S @ react-dom.production.min.js:16

A @ react-dom.production.min.js:17

x @ react-dom.production.min.js:17

D @ react-dom.production.min.js:17

jn @ react-dom.production.min.js:81

ce @ react-dom.production.min.js:285

An @ react-dom.production.min.js:83

Tn @ react-dom.production.min.js:84

Dn @ react-dom.production.min.js:84

t.unstable_runWithPriority @ scheduler.production.min.js:18

Vr @ react-dom.production.min.js:120

se @ react-dom.production.min.js:285

xn @ react-dom.production.min.js:82

Show 33 more frames

Logger.js:154 2021-10-06T20:10:26.332Z [modules/xmpp/moderator.js] <l._allocateConferenceFocusError>: Focus error, retry after 1000

**Comment by damencho** [comment]
Yes, asks your questions in the community.jitsi.org, as those maybe not issues/bugs with the software but just configuration problems ...
